Library/Homebrew/rubocops/shared/url_helper.rb

```diff
@@ -241,7 +241,7 @@ def audit_url(type, urls, mirrors, livecheck_url: false)
         audit_urls(urls, zip_gh_pattern) do |_, url|
           next if url.match? %r{raw.githubusercontent.com/.*/.*/(main|master|HEAD)/}
           next if url.include?("releases/download")
-          next if url.include?("desktop.githubusercontent.com/github-desktop/releases/")
+          next if url.include?("desktop.githubusercontent.com/releases/")
```

Comment:
```suggestion
          next if url.include?("desktop.githubusercontent.com/github-desktop/releases/")
          next if url.include?("desktop.githubusercontent.com/releases/")
```
Keep this in while the cask is being updated, then remove it once it is.